prometheus/prometheus.yml

```diff
@@ -1,13 +1,24 @@
+rule_files:
```

Comment:
In production, this will be generated in chef and editable server side, but this local file is a good starting place to test configurations.